specs/commandline/sign.md

```diff
@@ -4,7 +4,21 @@
 
 Use `notation sign` to sign artifacts.
 
-Signs an OCI artifact that is stored in a registry. Upon successful signing, the generated signature is pushed to the registry with the digest of the OCI artifact returned.
+Signs an OCI artifact that is stored in a registry. Always use a `digest` to identify an artifact. `Tags` are mutable, but `digests` uniquely and immutably identify artifacts. If a tag is used, notation resolves the tag to the `digest` before signing.
+
+Upon successful signing, the generated signature is pushed to the registry and associated with the signed OCI artifact. The output message is printed out as following:
+
+```text
+Sign succeeded. Signature has been attached to <registry>/<repository>@<digest>.
```

Comment:
This can be improved. I would recommend following some guidelines (here a good example https://spark.apache.org/error-message-guidelines.html). I would recommend the following:

```Signing of artifact <artifact-as-provided-in-the-command> succeeded and the signature is attached to <registry>/<repository>@<digest>.```